pkgs/by-name/gh/ghstack/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "ghstack";
+  version = "v0.9.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "ezyang";
+    repo = "ghstack";
+    rev = "0c412bf";
+    hash = "sha256-VDsYIeFL8U5anUJ9KtWhEUeuoaO2qu5K7lSnGTjxNGs=";
+  };
+
+  nativeBuildInputs = [ python3.pkgs.poetry-core ];
+
+  propagatedBuildInputs = with python3.pkgs; [
```

Comment:
```suggestion
  build-system = [ python3.pkgs.poetry-core ];

  dependencies = with python3.pkgs; [
```

Reply:
Thank you for the suggestion. I've updated my commit.